oap-server/banyandb-java-client/src/main/java/org/apache/skywalking/banyandb/v1/client/PairQueryCondition.java

```diff
@@ -20,39 +20,288 @@
 
 import org.apache.skywalking.banyandb.v1.Banyandb;
 
-import static org.apache.skywalking.banyandb.v1.Banyandb.PairQuery.BinaryOp.BINARY_OP_EQ;
+import java.util.List;
 
 /**
  * PairQuery represents a query condition, including field name, operator, and value(s);
  */
-public abstract class PairQueryCondition {
-    protected String fieldName;
+public abstract class PairQueryCondition<T> extends FieldAndValue<T> {
+    protected final org.apache.skywalking.banyandb.v1.Banyandb.PairQuery.BinaryOp op;
 
-    protected abstract Banyandb.PairQuery build();
+    private PairQueryCondition(String fieldName, Banyandb.PairQuery.BinaryOp op, T value) {
```

Comment:
`org.apache.skywalking.banyandb.v1.Banyandb.PairQuery.BinaryOp` === `Banyandb.PairQuery.BinaryOp`, right? Please make the codes consistent, and avoid unnecessary full package name in codes.

Reply:
Fixed